site/app/config/parameters.yml.dist

```diff
@@ -27,11 +27,11 @@ parameters:
 
     # OAuth configuration: GitHub
     # Register at https://github.com/settings/applications
-    github_app_id:     ~
-    github_app_secret: ~
+    github_app_id:     GITHUB_OAUTH_ID
+    github_app_secret: GITHUB_OAUTH_SECRET
 
     # OAuth configuration: Google
     # Register at https://console.developers.google.com/
     # https://console.developers.google.com/project/librecores/apiui/credential/oauthclient
-    google_app_id:     ~
-    google_app_secret: ~
+    google_app_id:     GOOGLE_OAUTH_ID
+    google_app_secret: GOOGLE_OAUTH_SECRET
```

Comment:
ah I see. Then probably the easiest path forward is to add a comment before the dummy values to hint people to replace it, and leave the change as-is otherwise.